### ISSUE TITLE
Error with webpack 5

### ISSUE THREAD

**Issue by pmorenopina** [SassNinja/media-query-plugin#54]
Hi!!!

When i try to use this plugin with Webpack 5... error... =(

`Error: Compilation.chunks was changed from Array to Set (using Array method 'sort' is not possible)
    at Set.set.<computed> [as sort] (/Users/CODE/node_modules/webpack/lib/util/deprecation.js:117:10)
    at /Users/CODE/node_modules/media-query-plugin/src/plugin.js:165:45
    at /Users/CODE/node_modules/webpack/lib/Compilation.js:337:47
    at Hook.eval [as callAsync] (eval at create (/Users/deploy/CODE/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:54:1)
    at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (/Users/CODE/node_modules/tapable/lib/Hook.js:18:14)
    at cont (/Users/CODE/node_modules/webpack/lib/Compilation.js:1997:33)
    at /Users/CODE/node_modules/webpack/lib/Compilation.js:2043:9
    at /Users/CODE/node_modules/neo-async/async.js:2830:7
    at Object.each (/Users/CODE/node_modules/neo-async/async.js:2850:39)
    at Compilation.createChunkAssets (/Users/CODE/node_modules/webpack/lib/Compilation.js:3108:12)
    at /Users/CODE/node_modules/webpack/lib/Compilation.js:2038:13
    at /Users/CODE/node_modules/webpack/lib/Compilation.js:2192:5
    at /Users/CODE/node_modules/neo-async/async.js:2818:7
    at done (/Users/CODE/node_modules/neo-async/async.js:3522:9)
    at /Users/CODE/node_modules/neo-async/async.js:2830:7
    at done (/Users/CODE/node_modules/neo-async/async.js:2865:11)`

Is this plugin supported into version 5?

Thx!!

**Comment by SassNinja** [comment]
Hi @pmorenopina 

thanks for the issue and sorry for the late response – I've been really busy the past months

I'll try to take care of it the next week (PRs are always welcome 😉)

**Comment by Enchiridion** [comment]
@SassNinja I just found this plugin and it does exactly what I want, except for working with Webpack 5.  I'm getting the same error.  I'd buy you a coffee if it'd help get this fixed! Your plugin is the only one that does this correctly.  I tried the other two but couldn't get them to work with Webpack 5 either.  I saw someone submitted, then deleted their pull request for wp5 support, maybe that could help? https://github.com/SassNinja/media-query-plugin/pull/57
